### PR TITLE
Add example use case for Slack event integration.

### PR DESCRIPTION
**Pull request by WiNloSt**
This PR is for #470

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=488) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=488) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-nodejs?pullRequest=488) it.</sub>

**Comment by thinkingserious** [comment]
Thanks @WiNloSt!

Would you mind resolving the conflict?

With Best Regards,

Elmer

**Comment by WiNloSt** [comment]
Sure

**Comment by WiNloSt** [comment]
@thinkingserious rebased!

**Comment by mbernier** [comment]
@WiNloSt

We have not been able to merge your Pull Request, but because you are awesome - we wanted to make sure you could still get a SendGrid Hacktoberfest shirt.  

Please go **fill [out our swag form](https://dx.sendgrid.com/swag) before Nov 5th** and we will send the shirt! (We know that you might have tried this before and it didn’t work, sorry about that!)

### **You have till Nov 5th to fill out this form** in order to get the Hacktoberfest shirt!

Thank you for contributing during Hacktoberfest! We hope to see you in the repos soon! Just so you know, we always give away a SendGrid shirt for your first ever non-Hacktoberfest PR that gets merged.

File: packages/mail/USE_CASES.md [hunk 1]
```diff
@@ -472,3 +473,119 @@ Find more information about all of SendGrid's whitelabeling related documentatio
...
+## Slack event integration
+- Create your Slack app by going to https://api.slack.com/apps/new
+- Go to "Event Subscriptions" configuration page
+- Turns "Enable Events" option to "on"
+- Now we need an actual endpoint to put in "Request URL" input box for the verification, so you will need to create an app for receiving Slack webhook and make sure your app can be accessed from the Internet
+  - Use any Node.js server of your choice to create an endpoint which accept "POST" method
+  - You can make sure the request is actually from your Slack app by compare the value of "token" key from request body with your "Verification Token" in your app's "App Credentials" section. You could store the token value in your server's environment variable to make it secured.
+
+    example request body
+    ```
```
Comment (clee) [inline packages/mail/USE_CASES.md]: You can add the word 'json' immediately after the backticks here to tell GitHub that this is JSON-syntax code.

**Comment by thinkingserious** [comment]
Hello @WiNloSt,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**